src/DacpacTool/PackageBuilder.cs

```diff
@@ -40,7 +41,7 @@ public void AddReference(string referenceFile, string externalParts = null, bool
             Model.AddReference(referenceFile, externalParts, suppressErrorsForMissingDependencies);
         }
 
-        private void ValidateReference(string referenceFile)
+        private static void ValidateReference(string referenceFile)
```

Comment:
Can you explain why we're making this static? From what I can tell, it looks like we use this as an instance (i.e. `using var packageBuilder = new PackageBuilder(new ActualConsole());`). I did a quick search and don't see any calls to static methods.